.scss-lint.yml

```diff
@@ -1,147 +0,0 @@
-scss_files: 'app/assets/stylesheets/**/*.scss'
```

Comment:
should these changes be in a separate PR? There is a lot in the roadrunner app I would like for us to eventually change (ie. I couldn't help but notice that one of the gems adds Sinatra to our Gemfile.lock as dependency, that gem should probably be replaced by something else that doesn't do that for example.) 

But this change seems like it should be a in a separate PR.